conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -116,10 +141,14 @@ public void update(String s) {
     }
 
     private boolean isAdActive() {
-        return bitmovinPlayer.isAd();
+        return bitmovinPlayer.isAd() || ssai.isAdBreakActive();
     }
```

Comment:
This implicitly enables tracking of 
- PLAY_HEAD_TIME
- PLAYER_STATE
on the `ConvivaAdAnalytics`.

Reply:
When reporting the play head time, is the current format what the conviva backend expects or do we have to convert the time to a relative timestamp since the ad(break) started? 🤔

Reply:
Excellent question. I would assume no but let me double check.

Reply:
Unfortunately there is no documentation to be found about this. Let's leave it as-is for now as it makes the most sense for SSAI ads.